src/Command/RunCommand.php

```diff
@@ -159,73 +228,76 @@ protected function configure(): void
                 Container::DEFAULT_FORMATTER_NAME
             )
             ->addOption(
-                'git-diff-filter',
+                self::OPTION_GIT_DIFF_FILTER,
                 null,
                 InputOption::VALUE_REQUIRED,
                 'Filter files to mutate git `--diff-filter` options. A - only for added files, AM - for added and modified.',
```

Comment:
`--diff-filter`?

Reply:
I think that it is not applicable here because we have `diff-filter` only once in this command

Reply:
I mean in this file